### PR TITLE
HPCC-20005 Remove all instances of ActiveX control

### PR DESCRIPTION
**Pull request by GordonSmith**
Fixes HPCC-20005

Signed-off-by: Gordon Smith <gordonjsmith@gmail.com>

## Type of change:
- [x] This change is a bug fix (non-breaking change which fixes an issue).
- [ ] This change is a new feature (non-breaking change which adds functionality).
- [ ] This change improves the code (refactor or other change that does not change the functionality)
- [ ] This change fixes warnings (the fix does not alter the functionality or the generated code)
- [ ] This change is a breaking change (fix or feature that will cause existing behavior to change).
- [ ] This change alters the query API (existing queries will have to be recompiled)

## Checklist:
- [x] My code follows the code style of this project.
  - [ ] My code does not create any new warnings from compiler, build system, or lint.
- [x] The commit message is properly formatted and free of typos.
  - [ ] The commit message title makes sense in a changelog, by itself.
  - [ ] The commit is signed.
- [ ] My change requires a change to the documentation.
  - [ ] I have updated the documentation accordingly, or...
  - [ ] I have created a JIRA ticket to update the documentation.
  - [ ] Any new interfaces or exported functions are appropriately commented.
- [x] I have read the CONTRIBUTORS document.
- [x] The change has been fully tested:
  - [ ] I have added tests to cover my changes.
  - [ ] All new and existing tests passed.
  - [ ] I have checked that this change does not introduce memory leaks.
  - [ ] I have used Valgrind or similar tools to check for potential issues.
- [x] I have given due consideration to all of the following potential concerns:
  - [ ] Scalability
  - [ ] Performance
  - [ ] Security
  - [ ] Thread-safety
  - [ ] Premature optimization
  - [ ] Existing deployed queries will not be broken
  - [ ] This change fixes the problem, not just the symptom
  - [ ] The target branch of this pull request is appropriate for such a change.
- [x] There are no similar instances of the same problem that should be addressed
  - [ ] I have addressed them here
  - [ ] I have raised JIRA issues to address them separately
- [x] This is a user interface / front-end modification
  - [ ] I have tested my changes in multiple modern browsers
  - [ ] The component(s) render as expected

### CODE REVIEW

File: esp/src/eclwatch/templates/HPCCPlatformWidget.html [hunk 1]
```diff
@@ -94,10 +100,14 @@
...
                 <div data-dojo-type="hpcc.TableContainer">
                     <input id="${id}ShowBanner" title="${i18n.Enable}:" name="BannerAction" data-dojo-type="dijit.form.CheckBox" />
-                    <input id="${id}BannerContent" title="${i18n.BannerMessage}:" name="BannerContent" style="width:100%;" data-dojo-props="trim: true" data-dojo-type="dijit.form.Textarea" />
-                    <input id="${id}BannerColor" title="${i18n.BannerColor}:" name="BannerColor" style="width:100%;" value="red" data-dojo-props="trim: true" data-dojo-type="HPCCColorPicker" />
-                    <input id="${id}BannerSize" title="${i18n.BannerSize}:" name="BannerSize" style="width:100%;" value="4" data-dojo-props="trim: true" data-dojo-type="dijit.form.Textarea" />
-                    <input id="${id}BannerScroll" title="${i18n.BannerScroll}:" name="BannerScroll" style="width:100%;" value="2" data-dojo-props="trim: true" data-dojo-type="dijit.form.Textarea" />
+                    <input id="${id}BannerContent" title="${i18n.BannerMessage}:" name="BannerContent" style="width:100%;" data-dojo-props="trim: true"
```
Comment (miguelvazq) [inline esp/src/eclwatch/templates/HPCCPlatformWidget.html]: Format changes needed here in this PR?

File: esp/src/lws.config.js [hunk 2]
```diff
@@ -1,4 +1,5 @@
-const debugServerIP = "192.168.99.103";
```
Comment (miguelvazq) [inline esp/src/lws.config.js]: Did you mean to check in this configuration? Probably not, due to no new paths / mappings being added.

File: esp/src/src/Utility.ts [hunk 3]
```diff
@@ -91,10 +91,10 @@ export function espTime2SecondsTests() {
     }, this);
 }
 
-export function convertedSize (intsize: number): string {
-    const unitConversion = ["Bytes","KB","MB","GB","TB","PB","EB","ZB","YB"];
-    const x = Math.floor(Math.log(intsize)/ Math.log(1024));
-    return (intsize/ Math.pow(1024,x)).toFixed(2) + " " + unitConversion[x];
+export function convertedSize(intsize: number): string {
```
Comment (miguelvazq) [inline esp/src/src/Utility.ts]: I'm assuming your VSCode "format on save" is doing some of these format changes similar to the ones I mentioned above?

**Review by miguelvazq — CHANGES_REQUESTED** [review]
Minor questions.

**Comment by GordonSmith** [comment]
@Miguel I reverted the white space changes,

**Comment by miguelvazq** [comment]
Looks good.

**Pull request merged**